lib/puppet/pops/lookup/hiera_config.rb

```diff
@@ -323,7 +365,15 @@ def create_configured_data_providers(lookup_invocation, parent_data_provider)
     data_providers = {}
 
     [@config[KEY_BACKENDS]].flatten.each do |backend|
-      raise Puppet::DataBinding::LookupError, "#{@config_path}: Backend '#{backend}' defined more than once" if data_providers.include?(backend)
+      if data_providers.include?(backend)
+        first_line = find_line_matching(/[^\w]#{backend}(?:[^\w]|$)/)
```

Comment:
Is it ok that these character classes inverse match only the ASCII character set for word ([a-zA-Z0-9_]) and not the unicode? I.e., `\w` instead of `[[:word:]]`